pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc

```diff
@@ -1,21 +1,22 @@
 [id="previewing-modules-in-pantheon_{context}"]
-= Previewing modules in {ProductShortName}
+= Previewing {ContentTerm} in {ProductShortName}
 
-Preview a module that has been imported in {ProductShortName}. 
+[role="_abstract"]
+Preview {ContentTerm} that has been imported in {ProductShortName}. 
 
 .Prerequisites
 * You are logged in.
 +
-If you are not logged in, you can still see the rendered module if it is already published to the customer portal.
+If you are not logged in, you can still see the rendered {ContentTerm} if it is already published to the {PublishingPortal}.
 
-* The module to be previewed has been imported in {ProductShortName}.
+* The {ContentTerm} to be previewed has been imported in {ProductShortName}.
 
 .Procedure
 . Navigate to the link:{LinkToSearchPage}[Search page].
-. Click the name of the module you want to preview.
+. Click the name of the {ContentTerm} you want to preview.
 +
-If you do not see the module listed, search for the modules using the search box.
+If you do not see the {ContentTerm} listed, search for the {ContentTerm} using the search box.
 
-. Click *Preview* next to the version of the module you want to preview.
+. Click *Preview* next to the version of the {ContentTerm} you want to preview.
 +
-If the module is already published to the Customer Portal, you will see *View* instead of *Preview*.
\ No newline at end of file
+If the {ContentTerm} is already published to the {PublishingPortal}, you will see *View* instead of *Preview*.
```

Comment:
Avoid future tense unless absolutely necessary, and there's no need for future tense here:

If the {ContentTerm} is already published to the {PublishingPortal}, Pantheon displays *View* instead of *Preview*.

Reply:
done